src/tests/unit/tests/common/components/cards/__snapshots__/rule-resources.test.tsx.snap

```diff
@@ -3,7 +3,7 @@
 exports[`RuleResources renders with {
   url: 'test-url',
   guidanceLinks: [ [length]: 0 ],
```

Comment:
This file needed to be updated to pass on my machine. Not really sure why, since it seems unrelated to these changes

Reply:
Looks like the PR build unit tests are failing here. When I tried out this change on my machine, running unit tests generated new snapshots without the `(anonymous)`. I'm guessing something is out of sync? Was a `yarn install` missed, or is your `node --version` something other than `v12.16.3`?

Reply:
Node version issue. Homebrew installed 14.X on my mac. It works now that I forced it back to 12.X